src/tests/TokenTests.t.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+import "../TokenFuzzTests.sol";
+import "./SampleToken.sol";
+
+contract TokenTestsTests is TokenTests, TokenFuzzTests 
```

Comment:
Shouldn't be this way as README says?
```suggestion
contract TokenTestsTests is TokenFuzzTests 
```

Reply:
It is fine to leave this way if you prefer. Just found a bit weird in the README mentions it and then in the example test still include both. But I'm fine either way.